coords_transformer/point/service.py

```diff
@@ -0,0 +1,10 @@
+from .factory import create_from_x_y_zoom
+from .point import Point
+
+
+def create_point_from_image_pixel_coords(x_distance: int, y_distance: int, ctr_pt: Point, size: int, zoom: int) -> Point:
```

Comment:
[nits]
I think some people are confused about `x_distance`, `y_distance` and `ctr_pt`, so please add some comments about it.